charts/operator-wandb/charts/weave/templates/deployment.yaml

```diff
@@ -89,13 +87,39 @@ spec:
           volumeMounts:
             - name: cache
               mountPath: /vol/weave/cache
+
+        - name: {{ include "weave.fullname" . }}-cache-clear
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          command: ["python", "-c"]
+          args: 
+          - |
+            import time
+            from weave import cache
+            print("Starting clear cache job", flush=True)
+            while True:
+              curTime = time.strftime("%x, %X", time.localtime())
+              print("Clearing cache " + curTime, flush=True)
+              cache.clear_cache()
+              time.sleep({{ .Values.clearCache.hourInterval | default 2 }}*60*60)
+            
+          env:
+            - name: WEAVE_LOCAL_ARTIFACT_DIR
+              value: /vol/weave/cache
+
+            {{- include "weave.extraEnv" (dict "global" .Values.global "local" .Values) | nindent 12 }}
+            {{- include "wandb.extraEnvFrom" (dict "root" $ "local" .) | nindent 12 }}
+
+          volumeMounts:
+            - name: cache
+              mountPath: /vol/weave/cache
+
       volumes:
         - name: cache
-          persistentVolumeClaim:
-            claimName: {{ template "weave.fullname" . }}
+          emptyDir: 
+            sizeLimit: {{ .Values.cache.size | default "5Gi" }}
```

Comment:
5Gi feels too small, how much disk do we typically have on the nodes @nfoucha @jsbroks ? Can we do like 50-80Gb here?

Reply:
We typically have 100 GB volumes. 

Capacity:
  ephemeral-storage:           104845292Ki
Allocatable:
  ephemeral-storage:           95551679124

Reply:
Does 20gb sound good? thats what the pvc was at before

Reply:
yes